File: netdemo/src/main/java/com/example/netdemo/MainActivity.java
package com.example.netdemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.OkHttpClient;
import okhttp3.Response;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: jetpackdemo/src/main/java/com/example/jetpackdemo/MainActivity.kt
package com.example.jetpackdemo

import android.os.Build
import android.os.Bundle
import androidx.annotation.RequiresApi
import androidx.appcompat.app.AppCompatActivity
import androidx.lifecycle.ViewModelProvider

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        test()
    }


    @RequiresApi(Build.VERSION_CODES.M)
    override fun onResume() {
        super.onResume()
    }


    fun test() {
        val viewModelProvider = ViewModelProvider(this)
        val studentViewModel = viewModelProvider.get(StudentViewModel::class.java)
        println(studentViewModel.age)
    }

}